.github/workflows/update-version.yaml

```diff
@@ -0,0 +1,60 @@
+name: Update VERSION.TXT
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - development
+
+jobs:
+  update_version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get latest tag from main
+        id: get_latest_tag
+        run: |
+          git fetch origin --tags main
+          latest_tag=$(git describe --abbrev=0 --tags origin/main 2>/dev/null)
```

Comment:
FWIW, a command like this seems to work fairly quickly without needing to clone a repo or fetch latest refs from a branch. Might be useful, if we do similar in konflux, and want to avoid cloning the whole repo:

```
$ git ls-remote --tags --sort='v:refname' git@github.com:RedHatProductSecurity/rapidast.git | grep -v 'tags/v' | tail -1 | cut -d/ -f3
```

The "grep -v" part is just there because it seems we switched from using "v" prefix in our versions at some point, and it messes up the sorting.

Reply:
One minor issue with this approach is that it doesn't consider branch information, so the command returns any tag, not just those present on `main`. However, this shouldn't be a problem since we don't tag outside of main.